english/dream/fly/asia/asia.md

```diff
@@ -15,10 +15,9 @@ around.
 
 What exotic adventures await?
 
+[Travel to Japan](./japan/japan.md)
 
-
-
-
+[Wake up](/eating-walls/find-a-toilet/find-a-toilet.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.